kratos/python/add_processes_to_python.cpp

```diff
@@ -175,18 +175,23 @@ void  AddProcessesToPython(pybind11::module& m)
     .def("GetNeighbourIds",&FindGlobalNodalNeighboursForConditionsProcess::GetNeighbourIds)
     ;
 
-    py::class_<FindGlobalNodalElementalNeighboursProcess, FindGlobalNodalElementalNeighboursProcess::Pointer, Process>
-        (m,"FindGlobalNodalElementalNeighboursProcess")
-    .def(py::init([](const DataCommunicator& rDataComm, ModelPart& rModelPart) {
-        KRATOS_WARNING("FindGlobalNodalElementalNeighboursProcess") << "Using deprecated constructor. Please use constructor without DataCommunicator.";
-        return Kratos::make_shared<FindGlobalNodalElementalNeighboursProcess>(rModelPart);
-    }))
-    .def(py::init([](ModelPart& rModelPart) {
-        return Kratos::make_shared<FindGlobalNodalElementalNeighboursProcess>(rModelPart);
-    }))
-    .def("ClearNeighbours",&FindGlobalNodalElementalNeighboursProcess::ClearNeighbours)
-    .def("GetNeighbourIds",&FindGlobalNodalElementalNeighboursProcess::GetNeighbourIds)
-    ;
+    using FindGlobalNodalElementalNeighboursProcessType = FindGlobalNodalEntityNeighboursProcess<ModelPart::ElementsContainerType>;
+    py::class_<FindGlobalNodalElementalNeighboursProcessType, typename FindGlobalNodalElementalNeighboursProcessType::Pointer, Process>(m,"FindGlobalNodalElementalNeighboursProcess")
+        .def(py::init([](const DataCommunicator& rDataComm, ModelPart& rModelPart) {
+            KRATOS_WARNING("FindGlobalNodalElementalNeighboursProcess") << "Using deprecated constructor. Please use constructor without DataCommunicator.";
+            return Kratos::make_shared<FindGlobalNodalElementalNeighboursProcessType>(rModelPart);
+        }))
+        .def(py::init<ModelPart&>())
+        .def("ClearNeighbours",&FindGlobalNodalElementalNeighboursProcessType::ClearNeighbours)
+        .def("GetNeighbourIds",&FindGlobalNodalElementalNeighboursProcessType::GetNeighbourIds)
```

Comment:
I'm not sure if we really want to expose this... I don't see a clear use case in Python and it goes against the public `Process` API.

Reply:
I use/need this :)

Actually it is used in several places too

Reply:
OK. I don't really like but if it is already populated we need to live with that.